javascript/javascript3/week1/Weather/weatherApp.js

```diff
@@ -0,0 +1,34 @@
+const keyApi = "9829a96800b091a629aa8958c0e8b9c8";
+
+const searchBtn = document.getElementById("btn");
+searchBtn.addEventListener("click", () => {
+  const city = document.getElementById("city").value;
+  fetchData(city);
+});
+
+function fetchData(city) {
+  return fetch(
+    `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${keyApi}`
+  )
+    .then((response) => {
+      return response.json();
+    })
+    .then((data) => {
+      const logIcon = document.getElementById("icon");
+      const logoCode = data.weather[0].icon;
+      const icon = `http://openweathermap.org/img/wn/${logoCode}@2x.png`;
+      const infoToHTML = `<p>City: ${data.name}</p> 
+        <p>Overview: ${data.weather[0].main}</p> 
+        <p>Temperature: ${data.main.temp}</p> 
```

Comment:
This is not coding related, but maybe you want to add in what unit is your temperature and wind speed are.